scripts/flitcli/flit_bisect.py

```diff
@@ -412,6 +412,41 @@ def extract_symbols(file_or_filelist, objdir):
 
     return symbol_tuples
 
+def memoize_strlist_func(func):
+    '''
+    Memoize a function that takes a list of strings and returns a value.  This
```

Comment:
You say the function being memoized takes a list of strings, but your example function takes a list of integers.